### PR TITLE
refactor: add the job to index nft descriptions

### PR DESCRIPTION
**Pull request by crnkovic**
## Summary

https://app.clickup.com/t/862kg76yh

This PR adds a CLI command `nfts:update-descriptions`. This command accepts the network ID (initially: 1). When run, it will dispatch a job to update descriptions for NFTs in that network.

It will start for first 100 NFTs (ordered by ID). Batch update the description, then dispatch the same job for the next 100. Essentially, it will loop 100 by 100. When this network is finished, it will start for the next network. This reduces rate limits as it runs one-by-one. Even if rate limits happen, a job will wait until rate limit is cleared before proceeding.

 Locally, I have around 1M NFTs and it took ~2.5 hours to finish all jobs. As this needs to run only "once" and for existing NFTs, we can just run it over the weekend and not care about the duration.

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
missing coverage

**Review by oXtxNt9U — APPROVED** [review]

File: app/Jobs/UpdateNftDescription.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                                ->first();
+
+            if (! $nextNetwork) {
+                info('Done indexing descriptions!');
+
+                return;
+            }
+
+            // Start from first NFT...
+            self::dispatch(1, $nextNetwork);
```
Comment (ItsANameToo) [inline app/Jobs/UpdateNftDescription.php:59]: are we sure the minimum is 1? otherwise we will miss certain NFTs 🤔

Comment (ItsANameToo) [reply app/Jobs/UpdateNftDescription.php:59]:
```suggestion
            self::dispatch(0, $nextNetwork);
```
update: checked with demo dump and there are NFTs that start from 0

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: app/Jobs/UpdateNftDescription.php (hunk 1 above)
Comment (crnkovic) [reply app/Jobs/UpdateNftDescription.php:59]: there is NFT with ID = 0?

[the remark this answers is not in the document]
Comment (crnkovic) [reply app/Jobs/UpdateNftDescription.php:59]: ID in the database is sequential though...

[the remark this answers is not in the document]
Comment (oXtxNt9U) [reply app/Jobs/UpdateNftDescription.php:59]: Yeah, my bad I mistook it for the token id not the column id.

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**